rpc/executor.go

```diff
@@ -106,7 +106,7 @@ func (e *executor) Plan(in *pb.LoadRequest, stream pb.Executor_PlanServer) error
 	logger = logger.WithField("function", "executor.Plan")
 
 	if err := e.auth.authorize(ctx); err != nil {
-		logger.WithError(err).Info("authorization failed")
+		logger.WithError(err).Warning("authorization failed")
```

Comment:
On the server, it's a warning since a user failing authentication should be raised for audit purposes but it's a normal occurrence. It's (still) an error on the client side.